### PR TITLE
aap-4471 Fix service shape info

### PR DESCRIPTION
**Pull request by ariordan-redhat**
Addresses [AAP-4471](https://issues.redhat.com/browse/AAP-4471)

Affects `/titles/aap-on-azure`

[Section 1.4](https://access.redhat.com/documentation/en-us/red_hat_ansible_automation_platform/2.1/html/red_hat_ansible_automation_platform_on_microsoft_azure_guide/intro-to-aap-on-azure#con-aap-azure-infrastructure-usage_aap-azure-intro) of Ansible on Azure guide

### CODE REVIEW

File: downstream/modules/aap-on-azure/con-aap-azure-infrastructure-usage.adoc [hunk 1]
```diff
@@ -8,8 +8,8 @@ Azure Kubernetes service (AKS):: The Kubernetes cluster used to deploy {Platform
 +
 **Service Shape:**
 +
-* Compute nodes: Standard_D4s_v3 ((4 vCPUs x 16 GiB)
-* Autoscaling min nodes: 3
+* Compute nodes: Standard_D4s_v3 (4 vCPUs x 16 GiB)
```
[the remark this answers is not in the document]
Comment (oraNod) [reply downstream/modules/aap-on-azure/con-aap-azure-infrastructure-usage.adoc:11]: Nice one @ianf77 I do think those are different things. #353 is more for OCP (Operator docs) while this looks, to me, to align with sizing from AKS docs: https://docs.microsoft.com/en-us/azure/virtual-machines/dv3-dsv3-series#dv3-series

Note the 3rd column in the table there.

File: downstream/modules/aap-on-azure/con-aap-azure-infrastructure-usage.adoc [hunk 2]
```diff
@@ -8,8 +8,8 @@ Azure Kubernetes service (AKS):: The Kubernetes cluster used to deploy {Platform
...
+* Autoscaling min nodes: 2
```
[the remark this answers is not in the document]
Comment (oraNod) [reply downstream/modules/aap-on-azure/con-aap-azure-infrastructure-usage.adoc:12]: I would suggest replacing all instances of "min" and "max" with the full word for clarity (also helps with translation).

File: downstream/modules/aap-on-azure/con-aap-azure-infrastructure-usage.adoc (hunk 1 above)
[the remark this answers is not in the document]
Comment (ariordan-redhat) [reply downstream/modules/aap-on-azure/con-aap-azure-infrastructure-usage.adoc:11]: @ianf77 
GiB (Gibibyte) is the unit used for disk size in Azure 
1 GB = 1000³ bytes, but 1 GiB = 1024³ bytes.

File: downstream/modules/aap-on-azure/con-aap-azure-infrastructure-usage.adoc [hunk 3]
```diff
@@ -9,7 +9,7 @@ Azure Kubernetes service (AKS):: The Kubernetes cluster used to deploy {Platform
 **Service Shape:**
 +
 * Compute nodes: Standard_D4s_v3 (4 vCPUs x 16 GiB)
-* Autoscaling min nodes: 2
+* Autoscaling minimum nodes: 2
 * Autoscaling max nodes: 20
```
Comment (oraNod) [inline downstream/modules/aap-on-azure/con-aap-azure-infrastructure-usage.adoc]: Sorry @ariordan-redhat but could we change max to maximum here?

File: downstream/modules/aap-on-azure/con-aap-azure-infrastructure-usage.adoc (hunk 2 above)
[the remark this answers is not in the document]
Comment (ariordan-redhat) [reply downstream/modules/aap-on-azure/con-aap-azure-infrastructure-usage.adoc]: Done

**Review by oraNod — APPROVED** [review]

**Pull request merged**